recipes/daw_json_link/all/conanfile.py

```diff
@@ -12,70 +17,81 @@ class DawJsonLinkConan(ConanFile):
     homepage = "https://github.com/beached/daw_json_link"
     topics = ("json", "parse", "json-parser", "serialization", "constexpr", "header-only")
     settings = "os", "arch", "compiler", "build_type"
-    generators = "cmake", "cmake_find_package_multi"
     no_copy_source = True
     short_paths = True
 
-    _compiler_required_cpp17 = {
-        "Visual Studio": "16",
-        "gcc": "8",
-        "clang": "7",
-        "apple-clang": "12.0",
-    }
+    @property
+    def _min_cppstd(self):
```

Comment:
For consistency across other Conan packages.

```suggestion
    def _minimum_cpp_standard(self):
```